CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New features
 
 * [#5597](https://github.com/bbatsov/rubocop/pull/5597): Add new `Rails/HttpStatus` cop. ([@anthony-robin][])
+* [#5699](https://github.com/bbatsov/rubocop/pull/5699): Add `consistent_relative_to_receiver` style option to `Layout/FirstParameterIndentation` ([@jfelchner][])
```

Comment:
One failure is that this entry doesn't end with punctation. Adding a period will resolve it.

Reply:
I will see if I can give the meaningful failures a closer look this weekend. I've also been swamped and haven't been able to give `rubocop` the attention I'd like to over the past couple months.

Reply:
I was speaking more about the test failures.